internal/provider/provider.go

```diff
@@ -1,188 +1,189 @@
 package provider
 
 import (
-	"context"
-	"fmt"
-
-	"github.com/hashicorp/terraform-plugin-framework/datasource"
-	"github.com/hashicorp/terraform-plugin-framework/provider"
-	"github.com/hashicorp/terraform-plugin-framework/provider/schema"
-	"github.com/hashicorp/terraform-plugin-framework/resource"
-	"github.com/hashicorp/terraform-plugin-framework/types"
-
-	bloxoneclient "github.com/infobloxopen/bloxone-go-client/client"
-	"github.com/infobloxopen/terraform-provider-bloxone/internal/service/dns_config"
-	"github.com/infobloxopen/terraform-provider-bloxone/internal/service/dns_data"
-	"github.com/infobloxopen/terraform-provider-bloxone/internal/service/infra_mgmt"
-	"github.com/infobloxopen/terraform-provider-bloxone/internal/service/infra_provision"
-	"github.com/infobloxopen/terraform-provider-bloxone/internal/service/ipam"
-	"github.com/infobloxopen/terraform-provider-bloxone/internal/service/keys"
+    "context"
+    "fmt"
+
+    "github.com/hashicorp/terraform-plugin-framework/datasource"
+    "github.com/hashicorp/terraform-plugin-framework/provider"
+    "github.com/hashicorp/terraform-plugin-framework/provider/schema"
+    "github.com/hashicorp/terraform-plugin-framework/resource"
+    "github.com/hashicorp/terraform-plugin-framework/types"
+
+    bloxoneclient "github.com/infobloxopen/bloxone-go-client/client"
+    "github.com/infobloxopen/terraform-provider-bloxone/internal/service/dns_config"
+    "github.com/infobloxopen/terraform-provider-bloxone/internal/service/dns_data"
+    "github.com/infobloxopen/terraform-provider-bloxone/internal/service/infra_mgmt"
+    "github.com/infobloxopen/terraform-provider-bloxone/internal/service/infra_provision"
+    "github.com/infobloxopen/terraform-provider-bloxone/internal/service/ipam"
+    "github.com/infobloxopen/terraform-provider-bloxone/internal/service/keys"
```

Comment:
Why is the spacing shown to be different. `go fmt`?

Reply:
That's odd. Let me try go fmt again